### ISSUE TITLE
Chrome. Button "search on website".

### ISSUE THREAD

**Issue by duzers** [emvaized/selecton-extension#81]
Sometimes it works, but usually it doesn't.
Often it works so that I have to click the button several times.

**Comment by lattaai8** [comment]
Hello, I tried to solve the issue.

This is what I did:

Modified the addBasicTooltipButton, addContextualTooltipButton, and addLinkTooltipButton functions to use onclick instead of onmousedown and onmouseup. This should improve the consistency of button functionality.

You can review changes in this commit: https://github.com/lattaai8/emvaized-selecton-extension-81/commit/c7e2dc2a246d0246c49f2ed8b4871fe5612ada10.


> [!CAUTION]
> Disclaimer: The commit was created by Latta AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---
This issue was tried to solve for free by Latta AI - https://latta.ai/ourmission

If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by emvaized** [comment]
Thanks for your report! 
Please let me know if you're using latest version of Selecton (`3.9.4`). 

And also, does this happen only with "Search on website" button? Or any custom search button revealed when hovering over the "Search" button?

**Comment by duzers** [comment]
Yes, 3.9.4. Vivaldi Browser.

**Comment by emvaized** [comment]
I can not recreate this problem, so it is probably caused by some conflict with your other installed extensions.

**Comment by duzers** [comment]
I thought so too. I'll check it out.

**Comment by duzers** [comment]
I already know where the problem is. If the "Allow Text Selection in Links" option is enabled in Vivaldi, the search button does not always respond, it usually fails more often than it works.
<img width="510" alt="image" src="https://github.com/user-attachments/assets/2d273158-1e1a-45dc-861f-76e1780a04b2">

**Comment by emvaized** [comment]
Thanks for letting me know! 
Unfortunately, this is not the first time Vivaldi's built-in features breaks extension functionality. There's not much I can do about it, the only thing I can suggest is to disable this feature in Vivaldi, and switch to [Select Like A Boss](https://chromewebstore.google.com/detail/select-like-a-boss-select/mbnnmpmcijodolgeejegcijdamonganh?pli=1) extension, which Selecton is compatibile with.

**Comment by duzers** [comment]
No, this add-on makes it select all text from the bottom of the web page to the selection.

**Comment by emvaized** [comment]
> this add-on makes it select all text from the bottom of the web page to the selection.

That's not how it supposed to work. You probably have conflict with some another addon or built-in Vivaldi feature

**Comment by duzers** [comment]
> > this add-on makes it select all text from the bottom of the web page to the selection.
> 
> That's not how it supposed to work. You probably have conflict with some another addon or built-in Vivaldi feature

The problem may be more complicated than it seems. I transferred the entire profile to a virtual machine and Selection works fine there. It's possible that this is related to dual monitors (I recently discovered a bug related to this). I plan to reinstall the system in the future, so I will know more.
